docs/env-variables.md

```diff
@@ -0,0 +1,27 @@
+# Different Environment Variables used
+
+This page describes about the different environment variables that are
+used to configure kube-scheduler-simulator.
+
+`Port`: This is the port number on which kube-scheduler-simulator is started and it's value is 1212 which is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/3111ec419ccb8c49197b75385fbd166f7f159435/docker-compose.yml#L7)
+
+`EtcdURL`: This is the URL for kube-scheduler-simulator etcd which runs locally and it's value is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/3111ec419ccb8c49197b75385fbd166f7f159435/docker-compose.yml#L8)
+
+`FrontendURL`: This URL is on which kube-scheduler-simulator is reachable locally and it's value is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/3111ec419ccb8c49197b75385fbd166f7f159435/docker-compose.yml#L9).
```

Comment:
```suggestion
`FrontendURL`: This URL represents the URL web UI started.
The simulator and internal kube-apiserver set this origin as the allowed origin for CROS.
```

Reply:
Sorry, it's already renamed to CORS_ALLOWED_ORIGIN_LIST. Please rename
https://github.com/kubernetes-sigs/kube-scheduler-simulator/commit/c922c36d0f5678c9f5a9c9573aa333bbed91b43e

Reply:
Updated.